apps/system/js/nfc_handover_manager.js

```diff
@@ -161,6 +161,11 @@
       this.bluetoothStatusSaved = false;
       this.bluetoothAutoEnabled = false;
 
+      if (typeof(window.navigator.mozBluetooth.onattributechanged) !==
+        'undefined') {
+        this._debug('Bluetooth APIv2 does not support NFC yet');
+        return;
```

Comment:
So, you want to disable NFC module first. Then, turn it on in other follow up patch?